tools/postgres/diff_schema.js

```diff
@@ -6,15 +6,14 @@ const execSync = require("child_process").execSync;
 const path = require("path");
 const fs = require("fs");
 const glob = require("glob");
-const tmp = require("tmp");
 const rimraf = require("rimraf");
 const replace = require("replace-in-file");
 
 const POSTGRES_URL =
   typeof process.env.POSTGRES_URL !== "undefined"
     ? process.env.POSTGRES_URL
     : "jdbc:postgresql://localhost/webknossos";
-const scriptdir = __dirname;
+const scriptdir = "tools/postgres";
```

Comment:
we should rather use `__dirname` with a fallback (if I recall it correctly, the scriptName contains the full path in trireme)